robot/Cumulus/tests/browser/batch_gift_entry/bdi_batch_with_custom_fields.robot

```diff
@@ -0,0 +1,81 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Variables
+Suite Teardown  Delete Records and Close Browser
+
+*** Keywords ***
+Setup Variables
+    ${first_name1} =           Generate Random String
+    Set suite variable    ${first_name1}
+    ${last_name1} =            Generate Random String
+    Set suite variable    ${last_name1}
+    ${acc1}=                   Generate Random String 
+    Set suite variable    ${acc1}
+    ${first_name2} =           Generate Random String
+    Set suite variable    ${first_name2}
+    ${last_name2} =            Generate Random String
+    Set suite variable    ${last_name2}
+    ${acc2}=                   Generate Random String
+    Set suite variable    ${acc2}
+    ${org_ns} =  Get Org Namespace Prefix
+    Set suite variable    ${org_ns}
+    ${date} =     Get Current Date    result_format=%Y-%m-%d
+    Set suite variable    ${date}
+
+*** Test Cases ***
+
+Create Data Import Via API
+    [tags]  unstable
```

Comment:
As with the other PR, I just want to check on the unstable status. These tests passed on the PR run. Are they definitely unstable? Is there anything we can do to stabilize them?

Reply:
I'm going to go ahead and approve for the same reasons as the other PR. Let's continue to discuss!